tools/scylla-nodetool.cc

```diff
@@ -2644,6 +2645,241 @@ void table_stats_operation(scylla_rest_client& client, const bpo::variables_map&
+                sstring ident_delim = "";
+                for (const auto& child_info : el.GetObject()) {
+                    fmt::print("{}{}: {}", ident_delim, child_info.name.GetString(), child_info.value.GetString());
+                    ident_delim = ", ";
+                }
+                fmt::print(" }}");
+                delim = ", ";
+            }
+            fmt::print("]\n");
+        } else {
+            fmt::print("{} : {}\n", x.name.GetString(), x.value);
+        }
+    }
+}
+
+static void add_tree_to_statuses_lists(Tabulate& table, const rjson::value& res) {
+    auto statuses = res.GetArray();
+    for (auto& element : statuses) {
+        const auto& status = element.GetObject();
+
+        sstring children_ids = "[";
+        sstring delim = "";
+        if (status.HasMember("children_ids")) {
+            for (const auto& el : status["children_ids"].GetArray()) {
+                children_ids += format("{}{{", delim);
+                sstring ident_delim = "";
+                for (const auto& child_info : el.GetObject()) {
+                    children_ids += format("{}{}: {}", ident_delim, child_info.name.GetString(), child_info.value.GetString());
+                    ident_delim = ", ";
+                }
+                children_ids += format(" }}");
+                delim = ", ";
+            }
+        }
+        children_ids += "]";
+
+        table.add(rjson::to_string_view(status["id"]),
+                rjson::to_string_view(status["type"]),
+                rjson::to_string_view(status["kind"]),
+                rjson::to_string_view(status["scope"]),
+                rjson::to_string_view(status["state"]),
+                status["is_abortable"].GetBool(),
+                rjson::to_string_view(status["start_time"]),
+                rjson::to_string_view(status["end_time"]),
+                rjson::to_string_view(status["error"]),
+                rjson::to_string_view(status["parent_id"]),
+                status["sequence_number"].GetUint64(),
+                status["shard"].GetUint(),
+                rjson::to_string_view(status["keyspace"]),
+                rjson::to_string_view(status["table"]),
+                rjson::to_string_view(status["entity"]),
+                rjson::to_string_view(status["progress_units"]),
+                status["progress_total"].GetDouble(),
+                status["progress_completed"].GetDouble(),
+                children_ids);
+    }
+}
+
+static void print_trees(const std::vector<rjson::value>& res) {
+    Tabulate table;
+    table.add("id", "type", "kind", "scope", "state",
+        "is_abortable", "start_time", "end_time", "error", "parent_id",
+        "sequence_number", "shard", "keyspace", "table", "entity",
+        "progress_units", "total", "completed", "children_ids");
+
+    for (const auto& res_el : res) {
+        add_tree_to_statuses_lists(table, res_el);
+    }
+
+    table.print();
+}
+
+static void print_stats_list(const rjson::value& res) {
```

Comment:
These functions have names that are visible in the entire file, it would be good to add a `task` prefix/suffix, so they don't class with other function names.

Also, the `static` is not needed, all this code is wrapped in anonymous namespace (so all methods are implicitly static).